plugins/pulp_rpm/plugins/distributors/iso_rsync_distributor/publish.py

```diff
@@ -0,0 +1,103 @@
+import os
+
+from pulp.plugins.rsync.publish import Publisher, RSyncPublishStep
+from pulp.plugins.util.publish_step import RSyncFastForwardUnitPublishStep
+from pulp.server.db.model import Distributor
+from pulp.server.exceptions import PulpCodedException
+
+from pulp_rpm.common import constants, ids
+from pulp_rpm.plugins import error_codes
+
+class ISORsyncPublisher(Publisher):
```

Comment:
This will fail PEP8. 2 lines are needed before a class definition. Please check Jenkins and fix all flake8 errors.